openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/CubeOpenShiftConfiguration.java

```diff
@@ -140,47 +135,4 @@ public static CubeOpenShiftConfiguration fromMap(Configuration c, Map<String, St
             }
         }
     }
-
-    public static CubeOpenShiftConfiguration fromMap(Map<String, String> map) {
```

Comment:
but you are removing this method (fromMap from CubeOpenShiftConfiguration) completely which is used in `CubeOpenShiftConfigurator` and I don't see any new addition.